tests/test_jx/test_edge_1.py

```diff
@@ -1758,17 +1762,17 @@ def test_edge_w_partition_filters(self):
             "expecting_list": {
                 "meta": {"format": "list"},
                 "data": [
-                    {"a": "b", "count": 3, "sum": 15},
-                    {"a": "3", "count": 4, "sum": 37},
+                    {"a": "b", "count": 4, "sum": 28},
+                    {"a": "3", "count": 3, "sum": 24},
```

Comment:
This is wrong.   This is the test data:

    structured_test_data = [
        {"b": {"r": "a", "d": 1}, "v": 1},
        {"b": {"r": "a", "d": 2}, "v": 2},
        {"b": {"r": "a", "d": 3}, "v": 3},
        {"b": {"r": "b", "d": 1}, "v": 4},
        {"b": {"r": "b", "d": 2}, "v": 5},
        {"b": {"r": "b", "d": 3}, "v": 6},
        {"b": {"r": "c", "d": 1}, "v": 7},
        {"b": {"r": "c", "d": 2}, "v": 8},
        {"b": {"r": "c", "d": 3}, "v": 9},
        {"b": {"r": "d", "d": 1}, "v": 10},
        {"b": {"r": "d", "d": 2}, "v": 11},
        {"b": {"r": "d", "d": 3}, "v": 12},
        {"b": {"r": NULL, "d": 3}, "v": 13}
    ]

the only ones that match are:

        {"b": {"r": "b", "d": 1}, "v": 4},
        {"b": {"r": "b", "d": 2}, "v": 5},
        {"b": {"r": "b", "d": 3}, "v": 6},